pages/gallery.html

```diff
@@ -4,9 +4,54 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Beautiful Washington</title>
+  <link href="/styles/style.css" rel="stylesheet"> 
+  <link href="/styles/gallery.css" rel="stylesheet"> 
 </head>
+
 <body>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="index.html">Homepage</a></li>
+        <li><a href="gallery.html">Beautiful Washington</a></li>
+        <li><a href="facts.html">Camping in Washington</a></li>
+    </ul>
+    </nav>
+    <h1>Camping is Life</h1>
+  </header>
+  
+  <main>
+    <!-- Your main content goes here -->
+
+    <!-- Image 1 -->
+    <figure>
+        <img src="/assets/EastWinter.jpeg" alt="Eastern Washington in the Winter">
+        <figcaption>
+            Look how beautiful Eastern Washington is in the winter time!
+        </figcaption>
+    </figure>
+
+    <!-- Image 2 -->
+    <figure>
+        <img src="/assets/lakecrescent.jpeg" alt="Washington in the Summer">
+        <figcaption>
+            Look how beautiful Washington is in the summer! Source:
+        </figcaption>
+    </figure>
```

Comment:
Great use of `<figure>` and `<figcaption>`! Minor, but the copy says "Source:" and doesn't point anywhere haha